Duplicati/Library/Modules/Builtin/SendHttpMessage.cs

```diff
@@ -393,7 +393,7 @@ protected override void SendMessage(string subject, string body)
             using HttpClientHandler httpHandler = new HttpClientHandler();
             HttpClientHelper.ConfigureHandlerCertificateValidator(httpHandler, m_acceptAnyCertificate, m_acceptSpecificCertificates);
 
-            using var client = new HttpClient(httpHandler);
+            using var client = HttpClientHelper.CreateClient(httpHandler);
```

Comment:
This does not set the timeout, which I think is correct.
Do we need a comment here?